src/lein_dependency_check/core.clj

```diff
@@ -18,14 +18,18 @@
 	   (prn "Reconfiguring log4j")
 	   (PropertyConfigurator/configure (.getPath config-file)))))
 
+(defmacro ^:private with-cve-db
```

Comment:
There is no need for the lein plugin to reference the cve db directly. This is entirely wrapped in the engine now. I'm pretty sure this could be removed.